### ISSUE TITLE
TraceParserGen: xxxTaskGuid not defined

### ISSUE THREAD

**Issue by dorian-apanel-intel** [microsoft/perfview#1351]
I'm trying to compile cs code generated by TraceParserGen with custom manifest.

First problem I see is that guids passed to RegisterTemplate() are not defied.
Then I found this comment in code
```
// as of today, the generated code won't compile because the task GUID is not defined
// TODO: define the xxxTaskGuid based on eventGUID attribute of <task> elements of the .man file
```

Is it going to be fixed?
It happens both if I define eventGUID attribute or not.

**Comment by brianrob** [comment]
@dorian-apanel-intel, this can be fixed, but wasn't an issue that I was aware of.  Would you be up for submitting a PR for this?

**Comment by dorian-apanel-intel** [comment]
Hi, I fixed it for myself.
I'm going to propose a PR soon.

**Comment by limulidae** [comment]
Is this fixed?

**Comment by brianrob** [comment]
I don't believe that this has been fixed.  Fixes to TraceParserGen have been held up behind a goal to create a test-bed for the tool first, as each time we take a change, we tend to break something.

**Comment by brianrob** [comment]
Fixed via #2308.
